# This message holds the status of a specific test run on the system
#

# test types
byte WARN=0
byte ERROR=1

string test_name
byte test_type
bool result
